build/Targets/VSL.Imports.targets

```diff
@@ -204,6 +204,16 @@
     <ProjectLanguage Condition="'$(MSBuildProjectExtension)' == '.vbproj' OR '$(Language)' == 'VB'">VB</ProjectLanguage>
     <ProjectLanguage Condition="'$(MSBuildProjectExtension)' == '.vcxproj' OR '$(Language)' == 'C++'">C++</ProjectLanguage>
   </PropertyGroup>
+
+  <!-- VSIX projects generate a temp project file which doesnt have any extension and needs to be built as C# projects. The .NET SDK
+       just imporrts Microsoft.Common.CurrentVersion.targets. So set LanguageTargets to import CSharp.targets if the 
+       extension is unknown -->
```

Comment:
@tannergooding This is probably what blocks WPF too. Setting LanguageTargets to $(MSBuildToolsPath)\Microsoft.CSharp.targets will likely get it passed the point of failing to build the temp proj.